src/features/deepnav.tsx

```diff
@@ -851,31 +842,24 @@ const handleNavigateKey = (ev: KeyboardEvent) => {
   }
 };
 
+const callback = (
+  // e: Event
+  ) => {
+  // const inputTarget = getInputTarget(e);
+  // if (inputTarget) {
+    // inputTarget.blur();
+  // } 
+  window.addEventListener("keydown", keyDownListener);
+  navigate();
+}
+
 const keyDownListener = (ev: KeyboardEvent) => {
-  if (
-    keyIsModifier(ev) ||
-    ev.ctrlKey ||
-    (ev.altKey && (isNavigating || !isHotKey(ev)))
-  ) {
-    return;
-  }
   if (isNavigating) {
     if (getInputTarget(ev)) {
       endNavigate();
     } else {
       handleNavigateKey(ev);
     }
-  } else if (isHotKey(ev)) {
-    const inputTarget = getInputTarget(ev);
-    if (ev.altKey || !inputTarget) {
-      ev.stopImmediatePropagation();
-      ev.preventDefault();
-      // Deslect input before navigating
-      if (inputTarget) {
-        inputTarget.blur();
-      }
```

Comment:
I'm struggling with replicating this behavior via the addCommand callback.

In the static hotkey version, this is handled with an even listener to grab the DOM element to .blur() on.

How can I grab and pass the currently selected DOM element to the addCommand callback?

Reply:
```
const callback = (
  e: Event
  ) => {
  console.log(`e: ${e}`);
  }
```
This just returns an array with the block reference or null.

Any suggestions on what direction to take @dvargas92495 ?

Reply:
You could use `document.activeElement` and check to see if it is a textarea

Reply:
Ah!  Nice.

I was working on something like:

```
const getHTMLElementFromUid = (uid: string) => {
  const element = document.querySelector(`[id*="${uid}"]`) as HTMLElement;
  ...
```

Reply:
That could get you in trouble - the same block could be on the screen multiple times (main page + sidebar)